doc/source/whatsnew/v1.3.0.rst

```diff
@@ -754,6 +754,7 @@ Indexing
 - Bug in :meth:`DataFrame.loc.__setitem__` when setting-with-expansion incorrectly raising when the index in the expanding axis contains duplicates (:issue:`40096`)
 - Bug in :meth:`DataFrame.loc` incorrectly matching non-boolean index elements (:issue:`20432`)
 - Bug in :meth:`Series.__delitem__` with ``ExtensionDtype`` incorrectly casting to ``ndarray`` (:issue:`40386`)
+- Bug in :meth:`MultiIndex.union` raising for an empty ``RangeIndex`` (:issue:`41234`)
```

Comment:
is this a bug vc 1.2.x? if not then no whatsnew is needed (instead add this issue number next to the other release note)